skins/tao/css/editor.css

```diff
@@ -201,247 +45,92 @@ editors:
-  content: "\e651"; }
-
-.cke_button__bidiltr_icon:before {
-  content: "\e652"; }
-
-.cke_button__specialchar_icon:before {
-  content: "\e653"; }
-
-.cke_button__sourcedialog_icon:before {
-  content: "\e654"; }
-
-.cke_button__newpage_icon:before {
-  content: "\e655"; }
-
-.cke_button__templates_icon:before {
-  content: "\e656"; }
-
-.cke_button__cut_icon:before {
-  content: "\e657"; }
-
-.cke_button__replace_icon:before {
-  content: "\e658"; }
-
-.cke_button__copy_icon:before {
-  content: "\e659"; }
-
-.cke_button__paste_icon:before {
-  content: "\e65a"; }
-
-.cke_button__selectall_icon:before {
-  content: "\e65b"; }
-
-.cke_button__pastetext_icon:before {
-  content: "\e65c"; }
-
-.cke_button__pastefromword_icon:before {
-  content: "\e65d"; }
-
-.cke_button__bold_icon:before {
-  content: "\e65e"; }
-
-.cke_button__italic_icon:before {
-  content: "\e65f"; }
-
-.cke_button__underline_icon:before, .cke_button__taounderline_icon:before {
-  content: "\e660"; }
-
-.cke_button__subscript_icon:before {
-  content: "\e661"; }
-
-.cke_button__superscript_icon:before {
-  content: "\e662"; }
-
-.cke_button__strike_icon:before {
-  content: "\e663"; }
-
-.cke_button__outdent_icon:before {
-  content: "\e664"; }
-
-.cke_button__indent_icon:before {
-  content: "\e665"; }
-
-.cke_button__blockquote_icon:before {
-  content: "\e666"; }
-
-.cke_button__creatediv_icon:before {
-  content: "\e667"; }
-
-.cke_button__justifyleft_icon:before {
-  content: "\e668"; }
-
-.cke_button__justifycenter_icon:before {
-  content: "\e669"; }
-
-.cke_button__justifyright_icon:before {
-  content: "\e66a"; }
-
-.cke_button__justifyblock_icon:before {
-  content: "\e66b"; }
-
-.cke_button__taomediamanager_icon:before, .cke_button__taoqtimedia_icon:before {
-  content: "\e670"; }
-
-.cke_button__textcolor_icon:before {
-  content: "\e682"; }
-
-.cke_button__bgcolor_icon:before {
-  content: "\e683"; }
-
-.cke_button__scayt_icon:before {
-  content: "\e684"; }
-
-.cke_button__radio_icon:before {
-  content: "\e69a"; }
-
-.cke_button__taoqtimaths_icon:before {
-  content: "\e6ac"; }
-
+/*
+Usage:
+- linear-gradient((color1, color2, color3)) - returns linear-gradient with evenly distributed colors,
+   if 3 colors used then the position of each will be 33,33%
+- linear-gradient((color1 0%, color2 30%, color3 80%)) - returns linear-gradient with manually distributed colors,
+   first param - color, second - position. Also you can use px or other valid units for set position.
+*/
+/* Do not edit */
 /* Reset for single elements, not their children. */
 .cke_reset {
-  /* Do not include inheritable rules here. */
-  margin: 0;
-  padding: 0;
-  border: 0;
-  background: transparent;
-  text-decoration: none;
-  width: auto;
-  height: auto;
-  vertical-align: baseline;
-  box-sizing: content-box;
-  -moz-box-sizing: content-box;
-  -webkit-box-sizing: content-box;
-  position: static;
-  -webkit-transition: none;
-  -moz-transition: none;
-  -ms-transition: none;
```

Comment:
Are you sure the browserlist is correct? Box sizing is prefixed for webkit and moz, but not for ms?

Reply:
Cleaned sourse scss from css-properties with prefix in this commit https://github.com/oat-sa/tao-core-ui-fe/pull/123/commits/e9293c08006bad95a19e70cf0af866eea4769d13
And postcss in oat-sa/tao-core-ui-fe use
"browserslist": [
        "extends @oat-sa/browserslist-config-tao"
    ]
https://github.com/oat-sa/tao-core-ui-fe/blob/d9c1eb9579efad06c2310fe3b981d8a5ec3f118a/package.json#L88